lib/rules/media-query-no-invalid/README.md

```diff
@@ -0,0 +1,57 @@
+# media-query-no-invalid
+
+Disallow invalid media queries.
+
+<!-- prettier-ignore -->
+```css
+@media not(min-width: 300px) {}
+/**    ↑
+ * This media query */
+```
+
+Media queries must be grammatically valid according to the [Media Queries Level 5](https://www.w3.org/TR/mediaqueries-5/) specification.
```

Comment:
I am unsure if this is the best way to explain this rule.
I know this feature too well on a deeply technical level, so it isn't easy for me to explain it well for CSS authors.

Suggestions to improve would be very welcome :)

Reply:
If I understand correctly this rule checks everything between `@media` and `{`, correct?

Reply:
Yes, but at a different level, or from a different perspective than other rules.

This rule doesn't check :
- feature names (`width`, `hover`, ...)
- feature values (`10px`, `more`, `dark`, ...)
- feature name and value pairs (`width: 10px`)

You could say it only checks the structure.

Reply:
Alright then, if I had to suggest something Id say add a reminder that list the rules that check those.
As a user Id expect this rule to call them internally.

Reply:
Surely, it might be helpful to provide such a reminder, e.g., `media-feature-name-no-unknown`, `media-feature-name-value-no-unknown`, etc.

Reply:
Good idea, listing the rules that cover other parts makes it clearer.

I've updated the rule documentation.